loc/translations-import/vscode-powerplatform.cs.xlf

```diff
@@ -48,11 +48,9 @@
       </trans-unit>
       <trans-unit id="pacCLI.authPanel.welcome.whenInteractiveSupported">
```

Comment:
Why were the translations for `pacCLI.authPanel.welcome.whenInteractiveSupported`, `pacCLI.authPanel.welcome.whenInteractiveNotSupported`, and `pacCLI.EnvOrSolutionTreeItem.toolTip` removed?

Reply:
Brought back the missing translations.